src/jllvm/class/ByteCodeIterator.cpp

```diff
@@ -94,6 +94,16 @@ requires(std::is_same_v<OpCode, SIPush>) ByteCodeOp parseSIPush(const char* byte
     return SIPush{offset, consume<std::int16_t>(bytes)};
 }
 
+template <class OpCode>
+requires(std::is_same_v<OpCode, MultiANewArray>) ByteCodeOp parseMultiANewArray(const char* bytes, std::size_t offset)
```

Comment:
Is this and the `ByteCode.def` changes unrelated changes (based on the PR description)

Reply:
This was in preparation for `multianewarray`, i can remove it.

Reply:
done